### PR TITLE
doc: Remove nnce not matching

### PR DESCRIPTION
**Pull request by qinqon**
**Is this a BUG FIX or a FEATURE ?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind enhancement

**What this PR does / why we need it**:
Currectly the NNCEs does not appear at not matching anymore, this change
remove that from the documentation.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by RamLavi — APPROVED** [review]
/lgtm

**Review by rhrazdil — CHANGES_REQUESTED** [review]
There is another leftover of Matching condition in yaml output here:
https://github.com/nmstate/kubernetes-nmstate/pull/746/commits/b6b551776f5d54953ca76adaba7d61cddb4010a0#diff-7b953ac5115413e057a5a418b9868b856f404913404e7740e5a84f004745b294R156 
Could you please remove it as well?

EDIT: 102-configuration.md @line 156

**Review by rhrazdil — APPROVED** [review]
Thank you!
/approve
/lgtm

File: vendor/github.com/moby/term/.gitignore [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # global .gitignore or .git/info/exclude see https://help.github.com/articles/ignoring-files
 .*
 !.github
-!.gitignore
+#!.gitignore
```
Comment (phoracek) [inline vendor/github.com/moby/term/.gitignore:5]: How did this get here?

Comment (qinqon) [reply vendor/github.com/moby/term/.gitignore]: I was using a different clone and it was dirty :-/

Fixed....

**Comment by rhrazdil** [comment]
/hold

The number of NNCEs in progressing doesn't match the `maxUnavailable`

https://github.com/nmstate/kubernetes-nmstate/pull/746/commits/b8f0adceb258c8618bdb4111ab48c2b7a8a6b75b#diff-7b953ac5115413e057a5a418b9868b856f404913404e7740e5a84f004745b294R462

**Pull request closed without merging**